crates/serverless/.env.example

```diff
@@ -1,15 +1,16 @@
-LAGON_ROOT_DOMAIN=lagon.dev
+LAGON_ROOT_DOMAIN=localhost:4000
 LAGON_REGION=local
 LAGON_ISOLATES_CACHE_SECONDS=60
 LAGON_LISTEN_ADDR=0.0.0.0:4000
 
 DATABASE_URL=mysql://root:mysql@localhost:3306/lagon
 REDIS_URL=redis://localhost:6379
 
-S3_REGION=
-S3_BUCKET=
-S3_ACCESS_KEY_ID=
-S3_SECRET_ACCESS_KEY=
+S3_ENDPOINT=http://localhost:9000
```

Comment:
I think the port should be `9002` instead? Similar to `packages/dashboard/.env.example`

Reply:
good catch. changed it